static/app/views/issueDetails/groupUserFeedback.tsx

```diff
@@ -70,6 +71,12 @@ function GroupUserFeedback() {
   return (
     <StyledLayoutBody hasStreamlinedUI={hasStreamlinedUI}>
       <Layout.Main fullWidth>
+        {hasStreamlinedUI && (
+          <FilterMessage>
+            {t('The feedback shown below is not subject to search filters.')}
+            <StyledBreak />
+          </FilterMessage>
+        )}
```

Comment:
we can check if it has been setup yet with this https://github.com/getsentry/sentry/blob/master/static/app/views/userFeedback/userFeedbackEmpty.tsx#L34